requirements/compile.py

```diff
@@ -0,0 +1,33 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+if __name__ == "__main__":
+    os.chdir(Path(__file__).parent)
+    os.environ["CUSTOM_COMPILE_COMMAND"] = "requirements/compile.py"
+    common_args = ["-m", "piptools", "compile", "--generate-hashes"] + sys.argv[1:]
+
+    subprocess.run(
+        [
+            "python3.9",
```

Comment:
I noticed we're tying ourselves to 3.9 a lot - any particular reason?

Reply:
No particular reason. We can also add Python 3.10.

Reply:
No worries, all good for now.